docs/data/charts/overview/overview.md

```diff
@@ -25,19 +25,23 @@ To modify the styling of charts you can rely on all the MUI styling tools, such
 
 ## Getting started
 
+:::warning
+Note we're using the `next` tag to download the latest v7 **alpha** version.
```

Comment:
Maybe make it `pre-release` instead of `alpha` to avoid the need to update it after we start `beta`? 🤔 
Or we should just have considered going with `canary` instead. 🙈 
```suggestion
Note we're using the `next` tag to download the latest v7 **pre-release** version.
```